OpenShift-Installation/Openshift-Installation.rst

```diff
@@ -478,7 +479,7 @@ The daemonset files are pre-populated using the values provided in the 'nodes' f
 Post Installation
 -----------------------
 
-1. If you are using any VSP release prior to 5.1.2, enable 'Underlay Support' and 'Address Translation Support' for the Openshift domain on VSD Architect.
+1. Enable 'Underlay Support' and 'Address Translation Support' for the Openshift domain on VSD Architect.
```

Comment:
Why do we need to enable PAT to Underlay? I thought we confirmed that things work irrespective of PAT enabled or disabled?